modules/network-observability-enriched-flows.adoc

```diff
@@ -3,13 +3,13 @@
 // network_observability/configuring-operator.adoc
 
 :_content-type: PROCEDURE
-[id="network-observability-enriched-flows-kafka_{context}"]
+[id="network-observability-enriched-flows_{context}"]
 = Export enriched network flow data
 
-You can send network flows to Kafka, so that they can be consumed by any processor or storage that supports Kafka input, such as Splunk, Elasticsearch, or Fluentd.
+You can send network flows to Kafka, IPFIX,or both at the same time. Any processor or storage that supports Kafka or IPFIX input, such as Splunk, Elasticsearch, or Fluentd, can consume the enriched network flow data.
 
 .Prerequisites
-* Installed Kafka
+* You have installed Kafka, if you choose to export to Kafka.
```

Comment:
```suggestion
* Your Kafka or IPFIX collector endpoint(s) are available from Network Observability flowlogs-pipeline pods.
```

Here Kafka or any IPFIX collector may even be outside of the cluster. It just need to be accessible and ready.

Reply:
Okay, I re-worked this basically just added what you suggest :) TY!